### DISCUSSION TITLE
injectStyle() and tailwindcss in dev vs prod env

### DISCUSSION THREAD

**Discussion by devbro1** [xyflow/xyflow#2331] [General]
I ran into a very random issue in how css code is loaded.

my environment:
react 17.2.0 created using create-react-app
using typescript
react-flow-renderer 10.3.12
postcss 8.4.13
tailwindcss 3.0.17

In my dev env I created
```
<Handle type="target" className="w-4 h-2 w-full h-full rounded-none" position={Position.Top} />
```

it results in:
```
<div data-nodeid="1" data-handlepos="bottom" class="react-flow__handle react-flow__handle-bottom nodrag w-4 h-2 w-full h-full rounded-none source connectable"></div>
```

In dev environment tailwind without postcss compressing css, width is applied from react-flow__handle react-flow__handle-bottom then from w-4 which is desired. 
In prod the order of css defintions are changed which results in react-flow__handle react-flow__handle-bottom width to be final value.

I narrowed it down to the order css defintions are loaded:
in dev react-flow css is loaded into style then tailwind css is loaded into the next style tag.

In prod all css files are put into a single css file and loaded into a link tag THEN react-flow css tags are loaded into style tags.

I am not as good with postcss but when I looked into the code injectStyle() literally adds css into head tag and does not provide anything to postcss to work on.

What solutions I may have to resolve this issue. any ideas would be greatly appreciated. At this point I am willing to accept hacky solution over any proper solution.

**Comment by moklick** [comment]
You could use the version without css and load it on your own https://reactflow.dev/docs/guides/theming/#import-bundle-without-styles . Does this work for you?

**Reply by devbro1** [reply]
yes that actually worked, thank you so much for guiding me in this.
